pom.xml

```diff
@@ -117,6 +118,18 @@
         </repository>
     </distributionManagement>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${junit.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
It seems weird to use `dependencyManagement` in a project with a single `pom.xml` and for a single dependency. What exactly is the reason?

Reply:
It allows us to import (note the use of `import`scope) the junit-bom and get consistent versions of all the junit dependencies. In this case both `junit-jupiter-api` and `junit-jupiter-params`. Checking its probably not a big deal for junit but comes in quite handy for things which have modules which move a bit more independently. 

Arguably overkill in this single pom but its just my default way (when available) for dealing with multi module imports from the same group.

Reply:
Ok, good to know. Thanks for the explanation.